Домашка/Program.cs

```diff
@@ -1,12 +1,67 @@
 ﻿using System;
 
-namespace Домашка
+namespace Календарь
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Input a year");
+            var year = int.Parse(Console.ReadLine());
+
+            int numberOfYear;
+            if (year >= 1984)
+                numberOfYear = (year - 1984) % 60;
+            else
+                numberOfYear = (year - 1984) % 60 + 60;
+            Console.WriteLine("{0} is the year of {1} {2}.", year, GetColor(numberOfYear / 12), GetAnimal(numberOfYear % 12));
```

Comment:
Рекомендую поправить (мою) грамматическую ошибку.
```suggestion
            Console.WriteLine("{0} is the year of а {1} {2}.", year, GetColor(numberOfYear / 12), GetAnimal(numberOfYear % 12));
```